kolibri/plugins/learn/assets/src/routes/classesRoutes.js

```diff
@@ -36,6 +36,12 @@ export default [
   {
     name: ClassesPageNames.CLASS_ASSIGNMENTS,
     path: '/classes/:classId',
+    props: route => {
+      const classId = route.params.classId;
```

Comment:
I thought of passing classId as a prop to make it accessible in the component which was not possible with `this.$route.params.classId`